installer/msi/build-msi.ps1

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2018-Present MongoDB Inc.
+<#
+.SYNOPSIS
+    Builds an MSI for the MongoDB ODBC driver.
+.DESCRIPTION
+    .
+.PARAMETER Arch
+    The architecture, x86 or x64.
+#>
+Param(
+  [string]$Arch,
+  [string]$VersionLabel
+)
+
+$ErrorActionPreference = 'Stop'
+
+$ProjectName = "Atlas SQL ODBC"
+$sourceDir = pwd
+$resourceDir = pwd
+$binDir = pwd
+$objDIr = ".\objs\"
+#$WixPath = "C:\wixtools\bin\"
+# for local building, most installations will be in the directory below
+$WixPath = "C:\Program Files (x86)\WiX Toolset v3.11\bin"
+$wixUiExt = "$WixPath\WixUIExtension.dll"
+
+if ($VersionLabel -eq "snapshot") {
```

Comment:
What do you mean? It  will be snapshot any time we don't do a tagged release. Am I missing something? 🤔

Reply:
Oh I see, that never should have been checked in. I thought I removed it. The installer with 0.0.0 will not working because Windows doesn't like 0.0.0.

Reply:
Yes, odd, that it does like 0.0.0. We never had an installer for 0.1.0, so this is Ok. A little funky maybe. But that's Ok.

Reply:
I think they don't treat 0.0.0 as a legitimate semvar.